src/connection_string.ts

```diff
@@ -1223,9 +1223,15 @@ export const OPTIONS = {
   pfx: { type: 'any' },
   secureProtocol: { type: 'any' },
   index: { type: 'any' },
-  // Legacy Options, these are unused but left here to avoid errors with CSFLE lib
-  useNewUrlParser: { type: 'boolean' } as OptionDescriptor,
-  useUnifiedTopology: { type: 'boolean' } as OptionDescriptor,
+  // Legacy options from v3 era
+  useNewUrlParser: {
+    type: 'boolean',
+    deprecated: 'useNewUrlParser is no longer used it can be removed'
```

Comment:
Just a little not for these 2 messages, but maybe something noting that it will be removed in the next major release?

"option is a no-op and will be removed in the next major release". Thoughts?

Reply:
_"useNewUrlParser has been a no-op since Node.js Driver version x.y.z and will be removed in the next major version"_ maybe

Reply:
_`useNewUrlParser` has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version_

Does this look good to y'all? I have a _very_ slight preference to not phrasing it as "no-op"